pkg/advisory/validate.go

```diff
@@ -517,6 +416,25 @@ func (opts ValidateOptions) validateIndexDiff(ctx context.Context, diff IndexDif
 	return errorhelpers.LabelError("invalid change(s) in diff", errors.Join(errs...))
 }
 
+func (opts ValidateOptions) validateIndexDiffForAddedEvents(events []v2.Event, packageName string) error {
```

Comment:
Do we have tests to validate this change ?

Reply:
Good callout... some of the behavior is implicitly asserted already, but it makes sense to add more explicit conditions — added in 7452370